MapPathGUI.py

```diff
@@ -129,7 +129,7 @@ def updateGrid(self):
         self.pathSet.add(curr_tile)
         lidar_data = self.generate_sensor.generateLidar(
             10, curr_tile.row, curr_tile.col)
-        self.gridEmpty.updateGridLidar(
+        self.gridEmpty.update_grid_tup_data(
```

Comment:
@kimmylin727 @kennycai0 This will crash when it is called, update parameters accordingly

Reply:
resolving